data/io.elementary.files.appdata.xml.in.in

```diff
@@ -58,6 +58,25 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="Next" date="Future" urgency="medium">
+      <description>
+        <p>New features:</p>
+        <ul>
+          <li>Initial git plugin support</li>
+          <li>Follow global history setting when restoring and saving tabs</li>
+        </ul>
+        <p>Other updates:</p>
+        <ul>
+          <li>Ensure tabs showing same folder share an Async object</li>
+          <li>Fix drag with secondary button on empty space</li>
+          <li>Fix missing hicolor icons </li>
```

Comment:
Looking at the issue and the pull request for this commit, maybe something more like:
```suggestion
          <li>Show custom media type icons for installed apps</li>
```
I'm not sure `hicolor` means anything to regular users